ceph.spec.in

```diff
@@ -1432,6 +1435,8 @@ fi
 if [ $1 -eq 1 ] ; then
 /usr/bin/systemctl start ceph-osd.target >/dev/null 2>&1 || :
 fi
+#make the /usr/lib/sysctl.d/90-ceph-osd-sysctl.conf work
+sysctl -p %{_prefix}/lib/sysctl.d/90-ceph-osd-sysctl.conf
```

Comment:
please and call `%sysctl_apply` and use the `sysctl -p` as a fallback, as this macro is not available in el7.0 but exists in el7.2 and up. also, please use full path of binary, like
```
%if 0%{?sysctl_apply}
    %sysctl_apply 90-ceph-osd-sysctl.conf
%else
    /usr/lib/systemd/systemd-sysctl %{_sysctldir}/90-ceph-osd-sysctl.conf >/dev/null 2>&1 || :
%endif
```

Reply:
ok,I will change like this

Reply:
compare to sysctl -p，sysctl_apply has any advantage？